### ISSUE TITLE
The green fades when resizing a custom area are missleading

### ISSUE THREAD

**Issue by timunie** [NickeManarin/ScreenToGif#692]
## The "Problem"
If one wants to resize the rectangle to capture by mouse the recorder opens a magnifier with a green cross / fade to show the actual mouse position. This is very useful to catch the exact pixel. The problem here is, that the green fade shows the mouse position, and not the corner of the rectangle. 

![image](https://user-images.githubusercontent.com/47110241/87577096-ac0a2800-c6d2-11ea-884c-9762cbfaa88a.png)

## A possible solution
- Either sync the green cross with the corner of the rectangle
- or display the selection rectangle in the magnifier as well
- or center the mouse exactly on the rectangles corner if the user hits mouse down

P.S.: I wanted to take a gif, but this didn't work. I hope one can understand my idea.

Happy coding
Tim

**Comment by NickeManarin** [comment]
Hi Tim,

The green crosshair and the mouse cursor are in sync with the selection rectangle.
So, the landing pixels of the green crosshair, is same landing pixels of the selection rectangle, and the capture will have an offset of 1 pixel (in each side).

In this example, the line is in the middle of 1 pixel, so the distance from the selection to the white pixel is 1 pixel.
Green crosshair -> 1 dark pixel -> white pixels
![image](https://user-images.githubusercontent.com/14798947/87578626-10b18c80-c6ab-11ea-9e17-ae5016bd8b70.png)

Here's the result, 1 pixel thick dark border in the top/left sides.
![image](https://user-images.githubusercontent.com/14798947/87578760-448cb200-c6ab-11ea-8879-11104d0b323b.png)

The capture has too 1 pixel thick dark border to the top/left.
![image](https://user-images.githubusercontent.com/14798947/87578850-6ab25200-c6ab-11ea-9277-973164be4018.png)

I'm going to see how I can improve the UX of that magnifier.

**Comment by timunie** [comment]
Wow what a fast answer.😲 Your image is exactly what I am loooking for but it didn't work kn my mashine. I will doublecheck this tomorrow if there is any setting or if I missed an update.

**Comment by timunie** [comment]
Hi @NickeManarin 
I know double-checked my version and it is up to date.

1. Here is my magnifier when dragging the resizing thumb. The green cross is exactly on the border i like: 
![image](https://user-images.githubusercontent.com/47110241/87674759-bc73de80-c776-11ea-831a-9088204e3bad.png)
Are you with me, that the selection rectangle should start on the gray border?

2. And here is the result when releasing the mouse button (I tried it several times to be sure I didn't move the mouse by accident):
![image](https://user-images.githubusercontent.com/47110241/87675038-27bdb080-c777-11ea-9006-0f2146dd47f2.png)

3. If I capture, my image is cutted:
![image](https://user-images.githubusercontent.com/47110241/87675183-520f6e00-c777-11ea-8b82-34e2d580fb82.png)

Note: I am running Windows 7, so if it is related to Win7 I will close this issue. Win7 is not supported by MS today, so you don't need to. 

Anyway ScreenToGif is the best screen recorder I found :+1:

**Comment by NickeManarin** [comment]
Your selection in the first image is a few pixels to the side...

**Comment by timunie** [comment]
You are right, that I grabbed the dragging thumb on the outer corner for demonstration purpose. But this may be what the normal user also does. I can fairly live with it, but wanted to give back something, at least if I have an improvement idea share it.
If you like to stick with the current implementation, it is fine. Then just close the issue. If you want me to have a look if I can make a PR, just give me a  for doing it.

**Comment by NickeManarin** [comment]
Ahh, so the problem is the user experience after the first drag-to-select (when the user is dragging the little squares), right?

**Comment by byzod** [comment]
Can confirm

This is an example image, I want to capture pixel inside the red rectangle (including red rectangle itself)
![t2z](https://user-images.githubusercontent.com/848607/87874201-85adfa80-c9fa-11ea-98fd-c09ef4fdcdb2.png)

So this is how I select it (it's correct, right?)
![t2-s1](https://user-images.githubusercontent.com/848607/87874223-ac6c3100-c9fa-11ea-8800-62e050767fd1.png)
![t2-s2](https://user-images.githubusercontent.com/848607/87874224-aece8b00-c9fa-11ea-89b7-339fe74b1661.png)

Then I got this in editor
![t2-s3](https://user-images.githubusercontent.com/848607/87874230-b68e2f80-c9fa-11ea-8638-bb0170984e4e.png)

The left up corner works as expected but lost one row and column at right bottom corner

**Comment by NickeManarin** [comment]
@byzod I believe this was fixed with v2.27.

**Comment by byzod** [comment]
No, and it don't allow me to select small area now
> ![A1](https://user-images.githubusercontent.com/848607/92919689-85166b00-f463-11ea-879c-d63b777563ee.gif)
